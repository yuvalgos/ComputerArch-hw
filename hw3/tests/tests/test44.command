./dflow_calc tests/test44.dat tests/test44.in p27 d164 d196 p161 p56 p64 p125 p117 p196 p70 d60 p161 d20 p68 p101 d89 p149 p143 p87 d110 p176 d11 d29 d108 d127 d96 d136 d159 d28 d165 p48 d4 d79 d74 p50 d88 p73 d164 p85 d38 p49 d168 p130 d191 p48 d118 p6 d9 p143 d10 p158 p81 p41 d149 p43 p18 p86 d182 p68 d160 d164 d196 d128 d195 d188 d29 d52 p57 p48 d54 p93 d24 p23 p52 d127 p173 p22 p184 d108 p159 p75 p126 d193 p94 p33 p37