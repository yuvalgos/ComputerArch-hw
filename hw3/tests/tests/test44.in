1 29 6 18
5 17 4 0
0 7 11 23
0 8 30 6
8 11 2 15
2 18 22 9
5 14 29 0
7 13 11 22
0 22 21 22
7 31 5 13
4 7 19 2
2 20 27 18
0 21 23 16
7 0 29 4
2 15 14 13
4 17 9 2
1 22 9 27
2 24 3 27
8 31 3 28
5 20 9 9
1 24 7 18
7 25 9 18
8 27 18 6
1 15 6 15
7 30 23 25
2 19 26 27
6 16 26 25
4 4 0 21
2 16 10 18
8 23 13 18
4 14 15 4
2 12 3 15
4 1 28 15
2 29 27 22
0 12 4 18
7 25 2 23
7 3 24 8
8 25 20 1
1 2 8 18
5 28 5 11
7 7 11 10
1 23 28 29
3 17 18 24
1 20 4 13
8 10 28 1
8 8 26 5
4 15 8 30
0 21 14 5
3 7 12 5
1 25 25 18
6 21 19 21
8 7 16 18
8 26 30 16
3 17 2 19
2 15 9 17
7 4 14 1
6 4 27 16
7 18 1 27
1 16 9 9
4 5 7 29
3 4 7 25
1 11 23 26
1 11 8 1
7 9 16 30
5 12 12 28
3 14 30 22
8 19 1 25
5 12 26 24
3 18 26 18
0 19 29 27
6 18 22 22
6 24 19 6
1 9 6 29
4 3 31 0
5 27 1 28
4 29 15 18
1 8 27 27
0 18 16 18
3 11 13 1
5 10 4 2
6 12 27 9
1 9 3 14
0 26 9 8
6 14 4 13
7 31 0 8
4 7 8 31
5 3 2 13
6 0 23 31
4 22 27 20
5 30 3 16
3 30 12 11
0 12 26 8
2 24 12 29
3 22 11 25
3 4 8 8
1 12 25 0
4 28 11 16
5 29 13 6
7 29 25 8
5 17 13 25
7 28 21 0
5 29 20 7
0 22 28 20
4 24 20 11
3 13 19 19
5 2 20 14
5 5 25 28
6 28 26 0
1 19 10 26
6 26 27 27
5 24 4 31
3 13 10 1
2 12 11 12
3 16 14 13
5 8 27 20
3 2 15 26
6 6 19 25
2 3 3 4
7 9 19 30
8 31 23 7
2 6 16 12
3 28 14 11
5 4 20 15
1 4 9 16
1 3 11 8
3 22 10 3
7 4 25 25
4 24 4 1
4 19 25 4
7 3 8 21
8 14 4 13
5 16 28 5
4 25 25 17
2 15 30 30
6 26 26 5
6 19 24 10
5 4 13 25
8 9 14 4
0 28 1 20
1 19 1 22
2 25 3 13
4 6 0 15
4 13 9 18
2 20 11 3
6 13 17 30
3 27 0 1
1 3 21 20
4 4 10 4
2 0 28 14
6 10 27 15
3 31 12 7
8 26 10 21
6 8 13 17
8 20 4 1
2 1 30 11
7 27 13 22
6 0 12 10
3 2 11 17
5 5 3 13
5 11 16 12
3 1 26 18
5 24 5 11
0 4 16 8
3 16 22 26
7 18 25 31
7 10 15 28
5 27 4 27
5 9 13 27
1 26 3 11
1 8 24 15
2 13 6 7
7 0 3 1
6 12 19 10
0 25 5 18
5 0 27 16
4 27 12 21
2 30 31 1
7 16 28 15
6 4 2 5
2 9 3 11
5 23 22 22
0 23 19 23
0 4 22 25
3 21 20 9
6 6 1 4
7 1 2 16
2 3 24 24
5 26 10 15
2 3 15 5
5 5 9 14
1 11 31 7
1 13 12 3
3 9 16 25
1 6 19 31
0 8 13 19
8 15 9 23
1 29 20 17
8 10 24 27
5 16 30 24
7 21 27 23
