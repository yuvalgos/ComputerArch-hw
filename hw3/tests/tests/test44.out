Reading the opcodes latency info from tests/test44.dat ... Got latency for 9 opcodes
Reading the program file tests/test44.in ... Found 200 instructions
getProgDepth()==207
getDepDepth(27)==22
getInstDeps(164)=={140,150}
getInstDeps(196)=={153,99}
getDepDepth(161)==171
getDepDepth(56)==28
getDepDepth(64)==36
getDepDepth(125)==107
getDepDepth(117)==84
getDepDepth(196)==161
getDepDepth(70)==3
getInstDeps(60)=={51,49}
getDepDepth(161)==171
getInstDeps(20)=={10,5}
getDepDepth(68)==80
getDepDepth(101)==107
getInstDeps(89)=={86,58}
getDepDepth(149)==120
getDepDepth(143)==106
getDepDepth(87)==96
getInstDeps(110)=={94,84}
getDepDepth(176)==167
getInstDeps(11)=={-1,5}
getInstDeps(29)=={7,5}
getInstDeps(108)=={79,82}
getInstDeps(127)=={126,32}
getInstDeps(96)=={78,58}
getInstDeps(136)=={111,132}
getInstDeps(159)=={131,112}
getInstDeps(28)=={-1,5}
getInstDeps(165)=={133,138}
getDepDepth(48)==35
getInstDeps(4)=={-1,-1}
getInstDeps(79)=={60,38}
getInstDeps(74)=={32,39}
getDepDepth(50)==63
getInstDeps(88)=={74,43}
getDepDepth(73)==23
getInstDeps(164)=={140,150}
getDepDepth(85)==96
getInstDeps(38)=={3,5}
getDepDepth(49)==84
getInstDeps(168)=={146,159}
getDepDepth(130)==114
getInstDeps(191)=={172,188}
getDepDepth(48)==35
getInstDeps(118)=={108,90}
getDepDepth(6)==3
getInstDeps(9)=={-1,7}
getDepDepth(143)==106
getInstDeps(10)=={-1,-1}
getDepDepth(158)==117
getDepDepth(81)==50
getDepDepth(41)==36
getInstDeps(149)=={145,133}
getDepDepth(43)==42
getDepDepth(18)==0
getDepDepth(86)==42
getInstDeps(182)=={125,173}
getDepDepth(68)==80
getInstDeps(160)=={151,77}
getInstDeps(164)=={140,150}
getInstDeps(196)=={153,99}
getInstDeps(128)=={49,126}
getInstDeps(195)=={192,181}
getInstDeps(188)=={133,158}
getInstDeps(29)=={7,5}
getInstDeps(52)=={24,28}
getDepDepth(57)==61
getDepDepth(48)==35
getInstDeps(54)=={-1,53}
getDepDepth(93)==87
getInstDeps(24)=={-1,21}
getDepDepth(23)==47
getDepDepth(52)==43
getInstDeps(127)=={126,32}
getDepDepth(173)==147
getDepDepth(22)==5
getDepDepth(184)==167
getInstDeps(108)=={79,82}
getDepDepth(159)==158
getDepDepth(75)==45
getDepDepth(126)==87
getInstDeps(193)=={139,150}
getDepDepth(94)==77
getDepDepth(33)==28
getDepDepth(37)==61
